### PR TITLE
create administration roles

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: en/developer_guide/navigation_panel.rst [hunk 1]
```diff
@@ -28,21 +32,107 @@ For the service to be recognised as a menu entry, it should be tagged with ``ope
...
+* id : merge of the "nav-" string and the name of the strategy as returned by
+  ``NavigationPanelInterface::getName()``, for instance "nav-user"
 * data-url : API url to be called to retrieve the data to be displayed when clicking the menu
 
 Specifics
 ---------
 
-It is possible to modify the order of the items in the panel by changing the return value of the ``NavigationPanelInterface::getWeight()`` method.
-The heaviest elements are displayed below the other ones.
+Ordening the menus
```
Comment (juchi) [inline en/developer_guide/navigation_panel.rst]: ordering ?

File: en/developer_guide/navigation_panel.rst [hunk 2]
```diff
@@ -28,21 +32,107 @@ For the service to be recognised as a menu entry, it should be tagged with ``ope
...
+         * @param ContainerBuilder $container
+         */
+        public function build(ContainerBuilder $container)
+        {
+            parent::build($container);
+            $container->addCompilerPass(new RoleCompilerPass());
+        }
+    }
+
+After clearing the cache, you will see the role ``ROLE_BAR`` display in the role list in the ``Group``
```
Comment (juchi) [inline en/developer_guide/navigation_panel.rst]: displayed

**Comment by nicolasThal** [comment]
@juchi pr updated

**Comment by juchi** [comment]
:+1:

**Comment by alavieille** [comment]
:+1:

**Comment by ngilain** [comment]
:+1:

**Pull request merged**